lib/Dialect/TritonGPU/Transforms/Pipeliner/MatmulLoopPipeline.cpp

```diff
@@ -879,6 +897,8 @@ createAsyncOps(scf::ForOp &forOp,
   llvm::MapVector<int, StageGroup> stageGroups;
 
   for (auto &[loadOp, info] : loadToInfo) {
+    if (dependsOnAnotherLoadOp(loadOp))
```

Comment:
The trouble was caused mostly by subbtype loads, e.g.,  bf16 load, which could not be vectorized thus resulted in a 2-byte cp.async instruction which isn't supported. Perhaps we could narrow it down based on data types.